apps/guild-import-job/src/nats.ts

```diff
@@ -0,0 +1,148 @@
+import {
+  AckPolicy,
+  connect,
+  JsMsg,
+  nanos,
+  NatsConnection,
+  StringCodec,
+} from 'nats';
+
+// TODO: How are streams created
+// TODO: How is pulling filtered
```

Comment:
Do we have answers for these?

Reply:
Yup. This can be feasable through creating consumer configuration for each job, guild & csv. 
https://nats-io.github.io/nats.deno/interfaces/ConsumerConfig.html#filter_subject